staging/prometheus-operator/templates/prometheus/ingressperreplica.yaml

```diff
@@ -6,7 +6,7 @@ apiVersion: v1
 kind: List
 metadata:
   name: {{ include "prometheus-operator.fullname" $ }}-prometheus-ingressperreplica
-  namespace: {{ $.Release.Namespace }}
+  namespace: {{ template "prometheus-operator.namespace" $ }}
```

Comment:
` namespace: {{ template "prometheus-operator.namespace" . }}`now we have `$` is this a consistency issue?

Reply:
upstream 🤷‍♂️  https://github.com/helm/charts/blob/master/stable/prometheus-operator/templates/prometheus/ingressperreplica.yaml#L9